install/install.md

```diff
@@ -10,23 +10,91 @@ description: |
+
+**[Environment Setup](#environment)** <br/>
+- [Conda](#conda)
+- [Docker](#docker)
+- [Pip](#pip)
+- [Within WSL2](#wsl2)
+	- [Conda](#wsl2-conda)
+	- [Docker](#wsl2-docker)
+	- [Pip](#wsl2-pip)
+- [Build From Source](#source)
+
+**[Next Steps](#next-steps)**  
 
 <hr/>
+<div id="selector"></div>
+
+# Install RAPIDS
+Use the selector tool below to select your preferred method, packages, and environment to install RAPIDS. Certain combinations may not be possible and are dimmed automatically. 
+
+{% include selector.html %}
+
+<div id="troubleshooting"></div>
+
+## Install Troubleshooting
+
+### **PiP Issues**
+<i class="fas fa-info-circle"></i> Infiniband is not supported yet. <br/>
+<i class="fas fa-info-circle"></i> These packages are not compatible with Tensorflow pip packages. Please use the [NGC containers](https://catalog.ngc.nvidia.com/orgs/nvidia/containers/tensorflow){: target="_blank"} or conda packages instead. <br/>
+<i class="fas fa-info-circle"></i> If you experience a “Failed to import CuPy” error, please uninstall any existing versions of cupy and install cupy-cuda11x. For example:
+```
+pip uninstall cupy-cuda115; pip install cupy-cuda11x
+```
+<br/>
+
+<i class="fas fa-info-circle"></i> The following error message indicates a problem with your environment: 
+```
+ERROR: Could not find a version that satisfies the requirement cudf-cu11 (from versions: 0.0.1, 22.10.0)
+ERROR: No matching distribution found for cudf-cu11
+```
+Check the suggestions below for possible resolutions:
+- The pip index has moved from the initial experimental release! Ensure the correct `--extra-index-url=https://pypi.nvidia.com`
+- Your Python version must be 3.8, 3.9, or 3.10.
+- RAPIDS pip packages require a recent version of pip that [supports PEP600](https://peps.python.org/pep-0600/){: target="_blank"}. Some users may need to update pip: `pip install -U pip`
+
 <br/>
-<div id="provision"></div>
 
-## 1. Provision System
+<i class="fas fa-info-circle"></i> Dask / Jupyter / Tornado 6.2 dependency conflicts can occur. Install jupyter-client 7.3.4 if the error below occurs: <br/>
+```
+    ERROR: pip's dependency resolver does not currently take into account all the packages that are installed. This behavior is the source of the following dependency conflicts.
+    jupyter-client 7.4.2 requires tornado>=6.2, but you have tornado 6.1 which is incompatible.
+```
+
+<br/>
+
+### **Conda Issues**
+<i class="fas fa-info-circle"></i> The dependency solver takes too long or never resolves: <br/>
+Update conda to use the new [libmamba solver](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community/){: target="blank"} or use [Mamba directly](https://mamba.readthedocs.io/en/latest/installation.html){: target="_blank"}. 
+
+<br/>
+
+### **Docker Issues**
+<i class="fas fa-info-circle"></i> Jupyter Lab is not accessible:<br/>
+If the server has not started or needs to be restarted / stop, use the included [start/stop script](#docker-startstop).
```

Comment:
FYI, this may need to change with 23.06. I believe we're changing the way Jupyter Lab launches in the new user containers.

Reply:
Noted